en/08_Changelogs/6.0.0.md

```diff
@@ -986,6 +986,14 @@ The following new methods have been added to facilitate this upgrade:
 
 We've upgraded the various Symfony dependencies from v6 to v7.
 
+### Support for PHPUnit 11 {#phpunit}
+
+PHPUnit 11 is now used for unit testing core and supported modules. We recommend upgrading to that version for your own projects and modules as well.
+
+Part of this upgrade includes a change to assertions expecting warnings, errors, and notices. See [asserting errors, warnings and notices](/developer_guides/testing/unit_testing/#asserting-errors-warnings-and-notices) for details.
```

Comment:
This first note is as per https://github.com/silverstripe/developer-docs/pull/583/files#diff-778f6037afe94d5adb84d1359c47a107c321c2c6b65dba8490ea070d8e917187